adoc/Private-Registry-Installation.adoc

```diff
@@ -1,22 +1,28 @@
+**** One for the Harbor UI/API.
+**** One for the Notary API (only if `Notary` will be enabled).
+** If using just a LoadBalancer, ensure that you have one of the following:
+*** A predefined external IP address that can be associated with the Load Balancer service used to expose the {spr} services.
+*** An FQDN value that, later on, can be mapped in the external DNS to the external IP address dynamically allocated to the Load Balancer service during installation.
+* For the Harbor UI/API and Notary API, choose between using auto-generated TLS certificates or providing your own custom TLS certificates. If using your own, have the certificates ready.
+* Choose the persistent storage back-end to store OCI artifacts: a {kube} `StorageClass`, or one of the external storage services available from public-cloud providers.
 * Verify that the target {kube} cluster provides the required `StorageClass`(es). A `StorageClass` with `ReadWriteMany` access mode is required to fully enable high-availability and scalability for the `registry` {spr} component, unless an external storage service is used to store OCI artifacts.
-* Choose between using an external or internal database service. The internal database service doesn't support high-availability and scalability and is therefore *not recommended for production*.
-The external database service needs to be prepared separately beforehand.
-* Choose between using an external or internal Redis service. Similar to the database case, the internal Redis service doesn't support high-availability and scalability and *is not recommended for production*.
-** If a public cloud managed Redis service is to be used as an external Redis, it needs to be prepared separately beforehand
-** Instructions on installing the SUSE `redis-ha` operator service are included in the {spr} installation steps covered this section, if this is the preferred choice
-* Optionally, prepare a GitHub personal authentication token to prevent rate-limiting problems with `trivy` downloading its vulnerability database
+* Choose between using an internal or internal database service.
+** The internal database service does not support high-availability and scalability, and is therefore *not recommended for production*.
+** If you use an external database service instead, prepare it separately beforehand.
+* Choose between using an internal or external Redis service. Similarly to the database:
+** The internal Redis service does not support high-availability and scalability and *is not recommended for production*.
+** And again, if you will use a external, public-cloud-managed Redis service, you must prepare it separately beforehand.
+** If you prefer to use the SUSE `redis-ha` operator service, installation instructions are included in the {spr} installation steps covered this section.
+* Optionally, prepare a GitHub personal authentication token, in order to prevent rate-limiting problems when `trivy` downloads its vulnerability database.
```

Comment:
Following the style guide, don't we usually avoid the usage of "in order" ?? 

* Optionally, prepare a GitHub personal authentication token, to prevent